src/components/NavToggle/NavToggle.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from "react";
+import { s } from "./";
+import { 
+	NavContext as Nav, 
+	ACTIONS 
+} from "COMPONENTS/Navigation/";
+
+export default function NavToggle(props){
+
+	const {
+		state: {
+			open: isOpen
+		},
+		dispatch
+	} = useContext(Nav);
+
+	const {
+		setExpanded //callback for Navigation comp
+	} = props;
+
+	//EVENT HANDLING
+	//---------------------------
+	function toggleOpen(e){
+		if(e) e.preventDefault();
+
+		setExpanded(!isOpen);
+
+		dispatch({
+			type: ACTIONS.OPEN_NAVIGATION,
+			value: !isOpen
+		});
+	}// toggleOpen
+
+	return (
+			<button 
+				className={ s.button }
+				role="switch"
+				aria-checked={ isOpen }
+				aria-controls="navigation__links"
+				onClick={ toggleOpen }
+			>
+				<span className={ s.labelContainer }>
```

Comment:
```suggestion
				<span className={ s.container}>
```
no need prepend it with `label` now that its in its own component